### DISCUSSION TITLE
Error in option file:  Could not open log file

### DISCUSSION THREAD

**Discussion by Rouneq** [kannibalox/pyrosimple#64] [General]
Pulled in full-example.rc. Other than changing `cfg.basedir` to match the path on my seedbox (you've seen examples of it previously), no other modifications of the provided config lines. Starting up rtorrent, it errors:

`rtorrent: Error in option file: ~/.rtorrent.rc:26: Could not open log file '/media/zdah/<username>/private/rtorrent/log/rtorrent-1716071606.log'.`

Line 26 is

`log.open_file = "log", (cfg.logfile)`

I noticed the part of the config to "Create instance directories" is located below this line. I've moved it up before opening the log to eliminate this error.

**Comment by kannibalox** [comment]
Thanks for the heads up, I've fixed that in the repo. As a quick aside, the full-example.rc doesn't do any tuning, so any of those settings should be copied over from your providers' config.
